AUTHORS.md

```diff
@@ -5,75 +5,105 @@ Creative Commons Attribution 4.0 International license.
-Panorama/Dimension Rift.png | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Night Sky1.png     | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Night Sky2.png     | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Planet1.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Planet2.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Planet3.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Sky1.png           | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Sky2.png           | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Sunset1.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Panorama/Sunset2.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
-Sound/Cancel1.wav           | (c) 2009 LokiF [CC0]                 | https://opengameart.org/content/gui-sound-effects
-Sound/Cursor1.wav           | (c) 2012 Kenney [CC0]                | https://opengameart.org/content/51-ui-sound-effects-buttons-switches-and-clicks
-Sound/Decision2.wav         | (c) 2016 celestialghost8 [CC0]       | http://opengameart.org/content/8bit-sfx
-Sound/Item2.wav             | (c) 2009 LokiF [CC0]                 | https://opengameart.org/content/gui-sound-effects
-System/Royal.png            | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System/System.png           | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System/SystemA.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System/SystemB.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System/SystemC.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System2/System2a.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System2/System2b.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-System2/System2c.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
-Title/Title1.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
-Title/Title2.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
-Title/Title3.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
-Title/Title4.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
+Type|Asset Name|Year|Authors and License|Authors' URL
+:-: | :-: | :-: | :-: | :-:
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
+`BattleWeapon` | Weapon.png | 2010 | russidan (Alephman) [CC-BY] | None
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
+`CharSet` | Actor1.png | 2010 | Marina Navarro Travesset [CC-BY] | https://muerteatartajo.blogspot.com
+`CharSet` | Object1.png | 2017 | Tom Lemmens, <br/>Blarumyrran [CC0] (original chest) | https://github.com/lemtom
+`CharSet` | Object2.png | 2010 | Verdant_Jack [CC0] | https://community.easyrpg.org/t/test-for-new-rtp/1067/8
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
```

Comment:
What is the purpose of these star symbols?

Reply:
By enabling the rich preview, it seems to be a folder separator.

![Captura de pantalla de 2022-03-28 02-25-22](https://user-images.githubusercontent.com/174885/160357352-e3ec4ca7-69bd-4d2a-b9c1-d155b31a7c61.png)

Maybe by making separated tables for each folder type to save a column instead is simpler?

Reply:
yeah maybe make one table per type.